environment.yml

```diff
@@ -3,15 +3,28 @@ channels:
   - conda-forge
   - usgs-astrogeology
 dependencies:
-  - python >= 3
+  - cmake>=3.10
```

Comment:
Why is cmake in here?

Reply:
If I remember correctly, the idea was: since many knoten notebooks use ale, we added ale dependencies so the env would work without adding ale's by hand. Is there a better way to to this?

Reply:
This is adding a handful of un-used dependencies into the main knoten environment to support the notebooks. I'd rather see a separate environment file for the notebooks.

Reply:
How does adding the above yml file as:  knoten/examples/notebooks.yml sound? Then we can just do a conda update --file examples/notebooks.yml